### DISCUSSION TITLE
What features should be introduced with "Logging"

### DISCUSSION THREAD

**Discussion by Mews** [indrajithi/tiny-web-crawler#36] [Q&A]
In the milestone issue (#24 ) "Logging" is listed as a milestone for v1
I'm just wondering what features would be a part of this? I already noticed the `logging` submodule being added, and I'd like to pick up this feature :)

**Comment by indrajithi** [comment]
I have updated the story. Basically we need to support coloured logging with different log levels using the python build in logging. And remove all print statement. 

It would also be best if we can add no print rule for the lint to pass.
